examples/application/evaluate_gaming_traffic.cc

```diff
@@ -153,6 +157,7 @@ main (int argc, char *argv[])
       NS_LOG_UNCOND ("Total received bytes: " << packetSink->GetTotalRx ()
                 << " ( " << packetSink->GetTotalReceivedPackets () << " packets)");
       NS_LOG_UNCOND ("Number of failed packets: " << gamingServer->GetTotFailedPackets ());
+      NS_LOG_UNCOND ("Average throughput:" << averageThroughput << " Mbps");
```

Comment:
```suggestion
      NS_LOG_UNCOND ("Average throughput:" << averageThroughput << " Mbps (if 0, the default application throughput is used)");
```

Reply:
Actually this is not right. here `averageThroughput` should be the default value, not zero.

Reply:
that's what I wrote

Reply:
The default throughput will be used if we don't specify the BitRate in the input. 
In all cases, the `averageThroughput` printed here will not be zero. 
I'm not getting your point by adding this sentence

Reply:
you're right, my bad